### ISSUE TITLE
Basic usage question

### ISSUE THREAD

**Issue by sunscreem** [claudiodekker/inertia-laravel-testing#6]
I've scaffolded a new Laravel 8 app with Jetstream/Inertia and I'm working out how to write some tests.

```php
    /** @test */
    public function a_random_test(){

        $response = $this->get(route('dashboard'));

        $response->assertInertia('dashboard');

    }
```
I'm getting the failure:

`The response is not a view.`

Seems like I'm missing something obvious?

**Comment by claudiodekker** [comment]
Hi @sunscreem,

I took a quick look at this, and as it turns out this is expected, because the response (in this case) is a redirect to the login page (as the dashboard requires authentication). Try doing something like this instead:

```php
/** @test */
    public function a_random_test()
    {
        // $user = User::factory()->create();
        $user = User::firstOrFail();

        $response = $this
            ->actingAs($user)
            ->get(route('dashboard'));

        $response->assertInertia('dashboard');
    }
```

By doing this, the call will now be authenticated as the given user, meaning that the dashboard can be reached and a valid response will be given. The assertion will still fail with the following error, but that's literally a 1-character fix:
```
Failed asserting that two strings are equal.
Expected :'dashboard'
Actual   :'Dashboard'
```

As for how I figured this out: I went a step back, and looked at the response object. By doing `dd($response->content());` I quickly noticed the redirect.

Hope this helps!
